tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -109,3 +116,59 @@ Verify Quick Starts Work As Expected When At Least One Step Is Skipped
     END
     Run Keyword And Continue On Failure     QuickStart Status Should Be    ${element}      In Progress
     Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}  exp_link_text=Continue
+
+External URLs Should Not Be Broken
+    [Documentation]     Go through a QuickStart and checks the status of all the external links
+    ${sideWindowButtons}=   Get WebElements   //button[@class='pf-c-wizard__nav-link']
+
+    ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+    @{href_list}=       Evaluate           [item.get_attribute('href') for item in $element_list]
+    IF  ${href_list}
+        Validate URls   ${href_list}
+    END
+    FOR    ${sideWindowButton}    IN     @{sideWindowButtons}
+            # Sometimes it's not able to click side window button so we are checking for false
+            ${status}   Run Keyword And Return Status   Click Element  ${sideWindowButton}
+            IF  ${status} == False
+                Click Button    Next
+            END
+
+            FOR    ${counter}    IN RANGE    5
+                Press Keys    NONE    TAB
+            END
+
+            ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+            @{href_list}=       Evaluate           [item.get_attribute('href') for item in $element_list]
+
+            IF  ${href_list}
+                Validate URls   ${href_list}
+            END
+    END
+
+Verify Brokern Links in Quickstart
+    [Documentation]     Clicks on al the quick start and verify links
+    [Arguments]    ${qucickStartElements}
+    ${quickStartCount}=   Get Length           ${qucickStartElements}
+    ${TitleElements}=     Get WebElements      //div[@class="pf-c-card__title odh-card__doc-title"]
+    FOR    ${counter}    IN RANGE     ${quickStartCount}
+        Log To Console    "\n++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
+        ${Title}=   Get Text          ${TitleElements[${counter}]}
+        ${Title}=   Split To Lines    ${Title}
+        Log To Console                ${Title[${0}]}
+        Click Element                 ${qucickStartElements[${counter}]}
+        Wait Until Element Is Visible  //button[@class='pf-c-wizard__nav-link']
+        External URLs Should Not Be Broken
+    END
+
+Get QuickStart Items
+        Wait Until Resource Page Is Loaded
+        ${quickStartElements}=     Wait for QuickStart to Load
+        [Return]    ${quickStartElements}
+
+Validate URls
+    [Documentation]   Validates  urls
```

Comment:
I'm almost sure we already have a keyword to check URLs, right @bdattoma ?

Reply:
yes, now you could use `URLs HTTP Status Code Should Be Equal To`. It takes the webelement, extract the href attribute and check the HTTP status @mwaykole

Reply:
ack added

Reply:
code is still using `Validate URls` keyword

Reply:
is it reflected now?